int32 OriginID
int32 TargetID
---
RelationshipMessage theRelationship